### PR TITLE
Briefcase operation documentation

### PR DESCRIPTION
**Pull request by austeja-bentley**
In this PR:
- Added documentation for Briefcase operations in both `@itwin/imodels-client-management` and `@itwin/imodels-client-authoring` packages

### CODE REVIEW

File: clients/imodels-client-management/src/operations/briefcase/BriefcaseOperationParams.ts [hunk 1]
```diff
@@ -4,18 +4,29 @@
  *--------------------------------------------------------------------------------------------*/
 import { CollectionRequestParams, IModelScopedOperationParams } from "../../base";
 
+/** Special value which is an alias for the current user id. */
```
Comment (robertas-kerpys) [inline clients/imodels-client-management/src/operations/briefcase/BriefcaseOperationParams.ts:7]: Special value to indicate the id of current user who is making the request

File: clients/imodels-client-management/src/operations/briefcase/BriefcaseOperationParams.ts [hunk 2]
```diff
@@ -4,18 +4,29 @@
...
 
+/** Url parameters supported in Briefcase list query. */
 export interface GetBriefcaseListUrlParams extends CollectionRequestParams {
+  /** Filters Briefcases with specific owner. See {@link ValidOwnerIdValue}. */
   ownerId?: ValidOwnerIdValue;
 }
 
+/** Parameters for get Briefcase list operation. */
 export interface GetBriefcaseListParams extends IModelScopedOperationParams {
+  /** Parameters that will be appended to the entity list request url that will narrow down or alter the results. */
```
Comment (robertas-kerpys) [inline clients/imodels-client-management/src/operations/briefcase/BriefcaseOperationParams.ts:24]: I think alter the results is not needed, it is a bit misleading in my opinion. We should just leave narrow down the results.

File: clients/imodels-client-management/src/operations/briefcase/BriefcaseOperations.ts [hunk 3]
```diff
@@ -25,6 +42,13 @@ export class BriefcaseOperations<TOptions extends OperationOptions> extends Oper
     }));
   }
 
+  /**
+   * Gets a single Briefcase by id. This method returns a Briefcase in its full representation. Wraps the
```
Comment (robertas-kerpys) [inline clients/imodels-client-management/src/operations/briefcase/BriefcaseOperations.ts:46]: Gets a single Briefcase by **its** id.

**Review by robertas-kerpys — CHANGES_REQUESTED** [review]

File: clients/imodels-client-authoring/src/operations/briefcase/BriefcaseOperationParams.ts [hunk 4]
```diff
@@ -4,14 +4,20 @@
...
 
+/** Parameters for acquire Briefcase operation. */
 export interface AcquireBriefcaseParams extends IModelScopedOperationParams {
+  /** Properties for the new Briefcase. */
   briefcaseProperties?: BriefcaseProperties;
 }
 
+/** Parameters for release Briefcase operation. */
 export interface ReleaseBriefcaseParams extends IModelScopedOperationParams {
+  /** Briefcase id to release. */
```
[the remark this answers is not in the document]
Comment (austeja-bentley) [reply clients/imodels-client-authoring/src/operations/briefcase/BriefcaseOperationParams.ts]: Fixed: https://github.com/iTwin/imodels-clients/blob/b3a0ba5c80c6c55204fa675ed18de31a703c9e36/clients/imodels-client-authoring/src/operations/briefcase/BriefcaseOperationParams.ts#L21

File: clients/imodels-client-authoring/src/operations/briefcase/BriefcaseOperationParams.ts [hunk 5]
```diff
@@ -4,14 +4,20 @@
...
 
+/** Properties that should be specified when acquiring a new Briefcase. */
 export interface BriefcaseProperties {
+  /** Name of the device which will hold the briefcase. */
   deviceName: string;
 }
 
+/** Parameters for acquire Briefcase operation. */
 export interface AcquireBriefcaseParams extends IModelScopedOperationParams {
+  /** Properties for the new Briefcase. */
```
[the remark this answers is not in the document]
Comment (austeja-bentley) [reply clients/imodels-client-authoring/src/operations/briefcase/BriefcaseOperationParams.ts]: Fixed: https://github.com/iTwin/imodels-clients/blob/b3a0ba5c80c6c55204fa675ed18de31a703c9e36/clients/imodels-client-authoring/src/operations/briefcase/BriefcaseOperationParams.ts#L15

File: clients/imodels-client-management/src/base/interfaces/apiEntities/BriefcaseInterfaces.ts [hunk 6]
```diff
@@ -4,23 +4,33 @@
...
+  /** Briefcase display name. */
   displayName: string;
 }
 
 export interface Briefcase extends MinimalBriefcase {
+  /** Briefcase id in a number format. Value corresponds to {@link MinimalBriefcase.id} property. */
   briefcaseId: number;
+  /** Id of the user which acquired the Briefcase. */
   ownerId: string;
+  /** Datetime string when of when the Briefcase was acquired. */
```
[the remark this answers is not in the document]
Comment (austeja-bentley) [reply clients/imodels-client-management/src/base/interfaces/apiEntities/BriefcaseInterfaces.ts]: Fixed: https://github.com/iTwin/imodels-clients/blob/b3a0ba5c80c6c55204fa675ed18de31a703c9e36/clients/imodels-client-management/src/base/interfaces/apiEntities/BriefcaseInterfaces.ts#L20

File: clients/imodels-client-management/src/base/interfaces/apiEntities/BriefcaseInterfaces.ts [hunk 7]
```diff
@@ -4,23 +4,33 @@
...
   acquiredDateTime: string;
+  /** Size of the Briefcase in bytes. */
   fileSize: number;
+  /** Name of the device which holds the Briefcase. */
   deviceName: string | null;
 }
 
+/** DTO for single Briefcase API response. */
```
[the remark this answers is not in the document]
Comment (austeja-bentley) [reply clients/imodels-client-management/src/base/interfaces/apiEntities/BriefcaseInterfaces.ts]: Fixed: https://github.com/iTwin/imodels-clients/blob/b3a0ba5c80c6c55204fa675ed18de31a703c9e36/clients/imodels-client-management/src/base/interfaces/apiEntities/BriefcaseInterfaces.ts#L28 https://github.com/iTwin/imodels-clients/blob/b3a0ba5c80c6c55204fa675ed18de31a703c9e36/clients/imodels-client-management/src/base/interfaces/apiEntities/BriefcaseInterfaces.ts#L33

File: clients/imodels-client-management/src/operations/briefcase/BriefcaseOperationParams.ts (hunk 1 above)
Comment (austeja-bentley) [reply clients/imodels-client-management/src/operations/briefcase/BriefcaseOperationParams.ts]: Fixed: https://github.com/iTwin/imodels-clients/blob/b3a0ba5c80c6c55204fa675ed18de31a703c9e36/clients/imodels-client-management/src/operations/briefcase/BriefcaseOperationParams.ts#L7

File: clients/imodels-client-management/src/operations/briefcase/BriefcaseOperationParams.ts (hunk 2 above)
Comment (austeja-bentley) [reply clients/imodels-client-management/src/operations/briefcase/BriefcaseOperationParams.ts]: Fixed (here and in other places where the same sentence is used): https://github.com/iTwin/imodels-clients/blob/b3a0ba5c80c6c55204fa675ed18de31a703c9e36/clients/imodels-client-management/src/operations/briefcase/BriefcaseOperationParams.ts#L24

File: clients/imodels-client-management/src/operations/briefcase/BriefcaseOperations.ts (hunk 3 above)
Comment (austeja-bentley) [reply clients/imodels-client-management/src/operations/briefcase/BriefcaseOperations.ts]: Fixed: https://github.com/iTwin/imodels-clients/blob/b3a0ba5c80c6c55204fa675ed18de31a703c9e36/clients/imodels-client-management/src/operations/briefcase/BriefcaseOperations.ts#L46

**Review by robertas-kerpys — APPROVED** [review]

**Pull request merged**